docs/products/postgresql/howto/monitor-database-with-datadog.md

```diff
@@ -84,8 +81,8 @@ Monitoring for your service.
 
 ## Related pages
 
--   Learn more about
-    [Datadog and Aiven](/docs/integrations/datadog).
--   Learn more about [Datadog Deep Database
-    Monitoring](https://www.datadoghq.com/product/database-monitoring/)
-    from their product page.
+- Learn more about [Datadog and Aiven](/docs/integrations/datadog).
+- [Expose PgBouncer statistics to Datadog](/docs/products/postgresql/howto/pgbouncer-stats-datadog).
```

Comment:
I recommend "PgBouncer monitoring with Datadog" analogous to our "Database monitoring with Datadog".